plugins/modules/grafana_dashboard.py

```diff
@@ -638,6 +653,7 @@ def main():
         dashboard_revision=dict(type="str", default="1"),
         overwrite=dict(type="bool", default=False),
         commit_message=dict(type="str"),
+        content=dict(type=dict_str),
```

Comment:
https://docs.ansible.com/ansible/latest/dev_guide/developing_program_flow_modules.html#argument-spec
```suggestion
        content=dict(type="raw"),
```
I think it's better, if not mandatory, to stick with the officially specified argument types. With `raw`, you can pass the content unvalidated and then perform the actual validation of the `content` immediately afterward.

Reply:
@rndmh3ro Maybe you can share your opinion :)